app/uk/gov/hmrc/uknwauthcheckerapi/errors/ApiErrorHandler.scala

```diff
@@ -36,10 +36,11 @@ class ApiErrorHandler extends HttpErrorHandler with Logging {
 
     Future.successful(
       statusCode match {
-        case FORBIDDEN              => ForbiddenApiError.toResult
-        case NOT_FOUND              => NotFoundApiError.toResult
-        case SERVICE_UNAVAILABLE    => ServiceUnavailableApiError.toResult
-        case UNSUPPORTED_MEDIA_TYPE => NotAcceptableApiError.toResult
+        case FORBIDDEN                => ForbiddenApiError.toResult
+        case NOT_FOUND                => NotFoundApiError.toResult
+        case SERVICE_UNAVAILABLE      => ServiceUnavailableApiError.toResult
+        case UNSUPPORTED_MEDIA_TYPE   => NotAcceptableApiError.toResult
+        case REQUEST_ENTITY_TOO_LARGE => RequestEntityTooLargeError.toResult
```

Comment:
Please can you move this above SERVICE_UNAVAILABLE. Let's keep the ordering lexicographical.